scripts/msct_image.py

```diff
@@ -660,6 +659,14 @@ def get_permutation_from_orientations(orientation_in, orientation_out):
 
         return perm, inversion
 
+    def get_orientation_3d(self):
+        status, output = sct.run("isct_orientation3d -i %s -get" % self.absolutepath, 0)
+
+        if status != 0:
+            sct.printv('ERROR in get_orientation.', 1, 'error')
+        orientation = output.split()[-1]
```

Comment:
You are replacing `sct_image`'s `get_orientation_3d()` by an internal function which does the same thing, but differently.
Considering a function whose contract would be to either fail or return a proper orientation codeword, both implementations make significant assumptions on `isct_orientation3d`'s unstructured output:
- Original code assumes that it ends with `Input image orientation : ` then what follows is the orientation codeword. If for some reason something was added, it would be returned as being part of the orientation codeword. Failure would happen somewhere further in the calling code.
- Your shorter code assumes that it ends with the orientation codeword. If for some reason something was modified and that becomes not true, it would return something else. Failure would also happen somewhere further in the calling code.

A good practice when retrieving data from unstructured output is to verify that there is some context around it. That way if someone ever changes:
```
-1 0 -0
-0 -1 -4.897e-12
-0 -4.897e-12 1

Input image orientation : LPI
```
to
```
Input image orientation: LPI
Matrix: ...
```
or
```
LPI
```
it would be caught, and it would be more obvious that the command output style has changed.